clients/extension/src/navigation/routes/index.tsx

```diff
@@ -16,6 +16,9 @@ import VaultSettingsPage from '@clients/extension/src/pages/popup/pages/vault-se
 import VaultsPage from '@clients/extension/src/pages/popup/pages/vaults'
 import { createHashRouter, Navigate } from 'react-router-dom'
 
+import { SetupFastVaultPage } from '../../components/setup/fast/SetupFastVaultPage'
```

Comment:
Consider updating the import path to:
`import { SetupFastVaultPage } from '@clients/extension/src/components/setup/fast/SetupFastVaultPage'
`
Using absolute imports makes it easier to move components around without breaking relative paths. It also improves readability and makes it clearer where modules are coming from.